src/main/java/uk/gov/hmcts/probate/services/invitation/model/Invitation.java

```diff
@@ -34,10 +34,14 @@ public String getFirstName() {
         return firstName;
     }
 
+    public void setFirstName(String firstName) { this.firstName = firstName; }
```

Comment:
Can you please just update the style to use carridge returns so it's consistence. I.e. 
public String getLastName() {
    return lastName;
}

Reply:
Done :)